docs/toc.yml

```diff
@@ -1289,30 +1289,21 @@ items:
     - name: Use search options to find text in your Word add-in
       href: word/search-option-guidance.md
       displayName: Word
-
-- name: Patterns and practices
+- name: Samples
   items:
-  - name: Overview
-    href: overview/patterns-and-practices-overview.md
+  - name: Samples overview
+    href: overview/office-add-in-code-samples.md
   - name: Open in Excel
     href: excel/pnp-open-in-excel.md
     displayName: Excel
-  - name: Batching custom function remote calls
-    href: excel/custom-functions-batching.md
-    displayName: Excel, Custom Functions
-  - name: Authentication
-    href: develop/authentication-pnp.md
-  - name: Handle high DPI and DPI scaling in your Office solution
-    href: /office/client-developer/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution
   - name: Office Add-ins community call
```

Comment:
Does the Community Call still belong under "Samples"?

Reply:
Putting in my oar here. Recently, we had the idea to move the Resources page out from under "Development lifecycle" and make it a top-level node (related internal task 5417576). Maybe we could make that a section with VSTO/COM and this community calls page included too? E.g.
- Resources
  - Overview (contains current Resources page content)
  - Community calls
  - VSTO/COM transition

Reply:
@ElizabethSamuel-MSFT I think that works. Can I create a separate PR for that change? Thanks!

Reply:
I think that should be fine, David